pages/common/noti.md

```diff
@@ -0,0 +1,15 @@
+# noti
+
+> Monitor a process and trigger a notification.
```

Comment:
- Is it a graphical notification, or a textual one?
 - Does it show only when the command finishes executing?

Reply:
Yes, it's graphical, using the default OS notification banner/popup

As far as I know it's supposed indicate a finished process, but I am pretty sure you could somehow plug it in the middle of a chain of commands (with some shell magic like `tee` perhaps?).

Should that, or parts of that, be mentioned in the description?

Reply:
Maybe not, since this is a tldr page. The basic usage should be fine.

Reply:
Then we should leave it like that